config.json

```diff
@@ -297,7 +297,12 @@
       "core": false,
       "difficulty": 5,
       "slug": "nucleotide-count",
-      "topics": null,
+      "topics": [
+        "conditionals",
+        "exception_handling",
+        "parsing",
+        "searching"
```

Comment:
These look like great suggestions! :) I think `maps` should definitely also be a topic since this is one of the few exercises where you have to use maps. And maybe also `integers`? And `strings`? What do you think? :)

Reply:
Yes, this all sounds good too! I can update it :smile: